internal/rangecoding/decoder.go

```diff
@@ -191,7 +189,7 @@ func (r *Decoder) getBits(n int) uint32 {
 //
 // https://datatracker.ietf.org/doc/html/rfc6716#section-4.1.2.1
 func (r *Decoder) normalize() {
-	for float64(r.rangeSize) <= math.Pow(2, 23) {
+	for r.rangeSize <= 1<<23 {
```

Comment:
What do you think of making `1>>23` a constant? When I was reading ffmpeg I didn't understand where this value came from at first. I love the optimization :)

```
// Value equal to 2^23
const twoToTwentyThird = 1 << 23
```

Reply:
I would use:
```
// maxRangeSize is ... It's equal to 2^23.
const maxRangeSize = 1 << 23
```

Reply:
@egonelbre LGTM! If you want to push commit directly to master (added you to Pion org) or open a PR with that I can merge right now.

Reply:
I'm currently updating the changes and update the PR, as it's easier at this point. It'll take me 5-10 min to fix the issues.

Reply:
done